highlight.io/pages/launch-week-2.tsx

```diff
@@ -99,7 +100,7 @@ const LaunchPage = () => {
 				</div>
 				<div className="flex flex-col items-center gap-16 w-full max-w-[1200px] mx-auto my-10 px-8">
 					<DayOne />
-					<HiddenDay title="Day 2: July 18th" />
+					<DayTwo />
```

Comment:
can we conditionally render this on july 18th in pacific time? I.e. after midnight pacific time on the 18th?

Reply:
ehh maybe not actuallly. that's ok.